src/styles/variables/_colors.scss

```diff
@@ -1,9 +1,109 @@
+// Black
+$off-black-9: rgba(0, 0, 0, 0.9);
+$off-black-73: rgba(0, 0, 0, 0.73);
+$off-black-5: rgba(0, 0, 0, 0.5);
+$off-black-3: rgba(0, 0, 0, 0.3);
 $black: #000000;
```

Comment:
Could be slightly better design to not include opacity variants in variables, and instead use `rgba($black, [opacity])` when we use an off-black value. Otherwise, we might run into the situation where we have a long list of variables for arbitrary opacity amounts.